Playbooks/ThinkstCanary-Alert-Ingestion/readme.md

```diff
@@ -0,0 +1,384 @@
+                    "alertDynamicProperties": []
+                },
+                "customDetails": {
+                    "additionalDetails": "AdditionalDetails_s",
+                    "alertType": "AlertType_s",
+                    "alertUrl": "AlertUrl",
+                    "canaryId": "CanaryID_s",
+                    "canaryIp": "CanaryIP_s",
+                    "clientName": "Flock_s",
+                    "alertDescription": "Description_s",
+                    "canaryTokenDetails": "Reminder_s",
+                    "alertDetailed": "Intro_s",
+                    "attackerIp": "SourceIP",
+                    "attackerSystem": "SourceSystem",
+                    "alertTimestamp": "Timestamp_s"
+                },
+                "entityMappings": [
+                    {
+                        "entityType": "IP",
+                        "fieldMappings": [
+                            {
+                                "identifier": "Address",
+                                "columnName": "SourceIP"
+                            }
+                        ]
+                    },
+                    {
+                        "entityType": "Host",
+                        "fieldMappings": [
+                            {
+                                "identifier": "HostName",
+                                "columnName": "SourceSystem"
+                            }
+                        ]
+                    },
+                    {
+                        "entityType": "IP",
+                        "fieldMappings": [
+                            {
+                                "identifier": "Address",
+                                "columnName": "CanaryIP_s"
+                            }
+                        ]
+                    },
+                    {
+                        "entityType": "Host",
+                        "fieldMappings": [
+                            {
+                                "identifier": "HostName",
+                                "columnName": "CanaryName_s"
+                            }
+                        ]
+                    }
+                ],
+                "sentinelEntitiesMappings": [
+                    {
+                        "entityType": "SentinelEntities",
+                        "fieldMappings": [
+                            {
+                                "identifier": "Entities",
+                                "columnName": "AlertUrl"
+                            }
+                        ]
+                    }
+                ]
+            }
+        }
+    ]
+}
+
+```
+
+### c. Configurations in Thinkst Canary
+Step 3: Configure your Canary Console.
+Head over to your Console Global Settings.
+
+<img src="https://help.canary.tools/hc/article_attachments/4517136881693/mceclip0.png"/>
```

Comment:
Please check this screen shot, looks like not valid, talking about canary console, but screen shot given is of azure portal.

Rest everything fine